src/store/epic/node.epic.ts

```diff
@@ -36,8 +36,25 @@ const setNodePositionEpic: AppEpic = (action$, _, { graph }) =>
     map(nodeActions.setNodePosition.success)
   )
 
+const deleteNodeEpic: AppEpic = (action$, _, { graph }) =>
+  action$.pipe(
+    filter(isActionOf(nodeActions.deleteNode.request)),
+    map(action => {
+      const node = graph.getNode(action.payload)
+      graph.removeNode(action.payload)
+      if (node !== undefined) {
+        return node.cytoscapeDefinitions
+      } else {
+        map(nodeActions.deleteNode.failure)
```

Comment:
this doesn't work. you have to either use throw/catch or return the resulting action from here directly instead of the seperate `map` step.